portfolio/src/main/webapp/script.js

```diff
@@ -28,12 +28,18 @@ function addRandomGreeting() {
 }
 
 function addMessage() {
-  fetch('/data').then(response => response.json()).then((message) => {
-    const messageList = document.getElementById('message-container')
+  fetch('/data').then(response => response.json()).then((messages) => {  
+    const messageList = document.getElementById('past-messages');
     messageList.innerHTML = '';
-    console.log(message)
-    for (var i =0; i < message.length; i++){ 
-      messageList.appendChild(createListElement(message[i]));
+    for (var i = 0; i < messages.length-1; i++){ 
```

Comment:
You could use forEach and create a lambda function:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach